lib/SymbolGraphGen/SymbolGraphGen.cpp

```diff
@@ -70,16 +70,18 @@ symbolgraphgen::emitSymbolGraphForModule(ModuleDecl *M,
   SmallVector<Decl *, 64> ModuleDecls;
   M->getDisplayDecls(ModuleDecls);
 
-  llvm::errs() << ModuleDecls.size()
-      << " top-level declarations in this module.\n";
+  if (!Options.QuietMessages)
+    llvm::errs() << ModuleDecls.size()
```

Comment:
Another way to implement this would be to create a stream that forwards its input to `llvm:errs()` only if `QuietMessages` is true. That way we don't have to guard the messages around `if` checks. I don't really mind either way though.

Reply:
Unless there's some "optional stream wrapper" type that exists somewhere, this implementation feels like the "smallest diff" way to implement this. Otherwise, i'd have to implement the wrapper myself.